script4_pack/script4

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# This script merges and sorts the contents of two text files lexicographically (case-insensitive).
+# Usage: ./script4 file1.txt file2.txt
+
+file1="$1"
+file2="$2"
+
+{ cat "$file1"; echo; cat "$file2"; } | sort -f
```

Comment:
1. Your script adds an extra line on top (because you added `echo`) - please delete it
2. Or just simplify to:`cat $file1 $file2 | sort - f`. `cat` accepts more than one argments.